brave/brave-core
# Remove note from Reset Rewards modal

Resolves https://github.com/brave/brave-browser/issues/28864

## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

inserted by @Miyayes.

1. Create fresh profile (against staging)
2. Build up some estimated earnings.
3. Go to reset your Rewards profile. Even though estimated earnings is non-zero, no "Note:" should appear.
